pandas/core/groupby/groupby.py

```diff
@@ -1460,8 +1460,138 @@ def describe(self, **kwargs):
     @Appender(_doc_template)
     def resample(self, rule, *args, **kwargs):
         """
-        Provide resampling when using a TimeGrouper
-        Return a new grouper with our resampler appended
+        Provide resampling when using a TimeGrouper.
+
+        Given a grouper the function resamples it according to a string
+        "string" -> "frequency".
+
+        See the :ref:`frequency aliases <timeseries.offset-aliases>`
+        documentation for more details.
+
+        Parameters
+        ----------
+        rule : str or Offset
+            The offset string or object representing target grouper conversion.
+        *args, **kwargs
+            For compatibility with other groupby methods. See below for some
+            example parameters.
+        closed : {‘right’, ‘left’}
+            Which side of bin interval is closed.
+        label : {‘right’, ‘left’}
+            Which bin edge label to label bucket with.
+        loffset : timedelta
+            Adjust the resampled time labels.
```

Comment:
These parameters are not in the signature, are they the possible `kwargs`? If that's the case, we can add them as a list in the `kwargs` description.

Reply:
OK, the are just some samples of the kwargs, as they were requested.